### ISSUE TITLE
etable giving error: nchar(ROWNAMES) : 'nchar()' requires a character vector

### ISSUE THREAD

**Issue by leucotheas** [lrberge/fixest#353]
Hello,

etable is giving an error when trying to output (any) model. I removed the package and installed the newest version from source (0.10.5) and the error continues. I run the following:

```R
library(fixest)
library(pander)
gravity_pois = fepois(Euros ~ log(dist_km) | Origin + Destination + Product + Year, trade)
etable(gravity_pois)
```

And I obtain the following error:

Error in nchar(ROWNAMES) : 'nchar()' requires a character vector

However, just running gravity_pois gives the correct output:

```R
Poisson estimation, Dep. Var.: Euros
Observations: 38,325 
Fixed-effects: Origin: 15,  Destination: 15,  Product: 20,  Year: 10
Standard-errors: Clustered (Origin) 
             Estimate Std. Error t value  Pr(>|t|)    
log(dist_km) -1.52787   0.115678 -13.208 < 2.2e-16 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1
Log-Likelihood: -7.025e+11   Adj. Pseudo R2: 0.764032
           BIC:  1.405e+12     Squared Cor.: 0.612021
```

**Comment by lrberge** [comment]
Hi, could you provide the `sessionInfo()`? The previous code runs smoothly on my machine (btw is `pander` needed?).

Thanks.

**Comment by leucotheas** [comment]
Hello Laurent, pander is not needed, I do not know why I was loading it (sorry!).

Here is the session info (error keeps popping up).

R version 3.6.0 (2019-04-26)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19043)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252    LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                           LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] fixest_0.10.5

loaded via a namespace (and not attached):
 [1] zoo_1.8-10          compiler_3.6.0      sandwich_3.0-1      tools_3.6.0         dreamerr_1.2.3     
 [6] Rcpp_1.0.8.3        nlme_3.1-139        grid_3.6.0          Formula_1.2-4       numDeriv_2016.8-1.1
[11] lattice_0.20-38

**Comment by lrberge** [comment]
Thanks for checking. That's what I add in mind: a base R issue (most likely with default behavior on matrix conversion or the sort that I didn't notice).

The easy fix is to update R. I guess you're on a server, would that be possible? 

I may try in the future to fix bugs with R < 4.0.0 (or make a higher version of R a requirement).

**Comment by erinkirwin** [comment]
Hello,


Thank you very much for this package. It's made some of my PhD analysis so much more efficient. Just adding that I am on a server that is tied to an old version of R, and I can't work around this issue either. If it's possible to fix it in the next update, that would be amazing.

Best wishes,

Erin

Reproducible example, just in case (borrowed from [here](https://stackoverflow.com/questions/75110771/split-fixed-effect-regression-error-in-esttable-output-nchar-requires-a))

rm(list=ls())
options(digits = 7, scipen = 999)

library(fixest)
library(wooldridge) # for Dataset

data(crime4)
m1 <- feols(crmrte ~ polpc | year,  data = crime4, split = "year")
etable(m1)

**Comment by lrberge** [comment]
Hi everyone and thanks again for the reports!
There were any bugs in old R versions, they should (hopefully) all be fixed in 0.12.1.
